cgmanifest.json

```diff
@@ -14698,16 +14708,6 @@
         }
       }
     },
-    {
```

Comment:
Looks like there's some merge weirdness here with the `jna` changes.

Reply:
This is fine, the tooling we have for updating the `cgmanifest.json` will always sort the entries as well. The `jna` change had some manual changes in it and they were out of order, so this got fixed here.